idutils/ext.py

```diff
@@ -54,6 +54,9 @@ def get_scheme_config_func():
 
 from .schemes import IDUTILS_PID_SCHEMES
 
+# when Python >=3.12, remove importlib_metadata and replace with:
+# from importlib.metadata import entry_points
```

Comment:
Have you tried that in 3.12? I don't think it worked for me.

Reply:
Tests are passing, and it should work because importlib_metadata >= 5 has the same implementation as Python 3.12.